plane/source/docs/qacro-mode.rst

```diff
@@ -0,0 +1,30 @@
+.. _qacro-mode:
+
+=========+
+QACRO Mode
+=========+
+
+QACRO (for acrobatic) is a quadplane mode for advanced users that provides rate
+based stabilization like :ref:`Copter ACRO <copter:acro-mode>`.  QACRO mode is best suited for aerobatic flight of a copter tailsitter.
+
+It is recommended to set parameter Q_TAILSIT_INPUT to FixedWing (1) for this mode. This selects plane style controls with the aileron stick controlling body frame roll rate and the rudder stick controlling body frame yaw rate.
```

Comment:
this bit should probably be down with the other tailsitter stuff. Does it abide by Q_TAILSIT_INPUT  or just assume its 1?

Reply:
thanks, that's just wrong... input_type does not apply to QACRO at all.  I hope I got the description right on the tailsitter page.
QACRO isn't restricted to tailsitters, so I think this belongs here, though.